unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -48,6 +58,10 @@ const mapStateToProps = (state, { lock }) => {
 
 export default connect(mapStateToProps)(KeyList)
 
+const KeyListWrapper = styled.div`
```

Comment:
instead of creating a new styled.div, perhaps you can put this inside the Table definition? I think this is OK because we don't export it.

Reply:
It does not feel semantically correct to do that. We can't mix up the table with pagination component. You might want to add more customizations to `Table` or `KeyListWrapper` in future as they serve different functionalities.